plugins/out_loki/loki.c

```diff
@@ -1568,6 +1605,12 @@ static void cb_loki_flush(struct flb_event_chunk *event_chunk,
                         FLB_LOKI_CT, sizeof(FLB_LOKI_CT) - 1,
                         FLB_LOKI_CT_JSON, sizeof(FLB_LOKI_CT_JSON) - 1);
 
+    if (compressed == FLB_TRUE) {
+        flb_http_add_header(c,
+                            FLB_LOKI_HEADER_CE, sizeof(FLB_LOKI_HEADER_CE) - 1,
+                            FLB_LOKI_CE_GZIP, sizeof(FLB_LOKI_CE_GZIP) - 1);
```

Comment:
We have a short-hand function to set up gzip content encoding type: [flb_http_set_content_encoding_gzip](https://github.com/fluent/fluent-bit/blob/e5f12f9161fdce8ea9edbd77afad298eb1bda733/src/flb_http_client.c#L1046)
Would you mind if you used the function here?